utils/messages.js

```diff
@@ -18,4 +18,34 @@ function showSuccessfulCreationMessage(projectDir) {
 `);
 }
 
-module.exports = { showSuccessfulCreationMessage };
+function showCustomDomainInfoMessage() {
```

Comment:
I think this is too much text. How about just:

```
Type a valid domain you own (e.g. myawesomeapp.com):
```

Then, after submitting, format the domain with: 

```
domain.replace(/^(https?:\/\/)?(www.)?/i, "").replace(/\/$/, "")
```

(removing starting http(s) and www and closing slash)

Then, if it still has a sub-domain (`domain.split(".").length > 2`), then and only then show an error: 

```
Remake doesn't support sub-domains at the moment (e.g. app.yourapp.com). You must use a root domain (e.g. yourapp.com)
```

Reply:
Then check if the domain is valid